src/System.Private.ServiceModel/tests/Scenarios/Binding/Custom/CustomBindingTests.4.0.0.cs

```diff
@@ -45,54 +45,4 @@ public static void DefaultSettings_Http_Text_Echo_RoundTrips_String()
             ScenarioTestHelpers.CloseCommunicationObjects((ICommunicationObject)serviceProxy, factory);
         }
     }
-
-    // Https: Client and Server bindings setup exactly the same using default settings.
-#if FULLXUNIT_NOTSUPPORTED
```

Comment:
Reviewers: this test was already broken in our "test shipped packages against latest toolchain" ToF tests. It was broken because X509Certificate2.Thumbprint was not available in those packages, making our ConditionalFact to check certificates throw exceptions.  It broke for a new reason with this PR (OSPlatform API's).  Therefore I have moved it to the non-4.0.0.cs file.